Aptoide/aptoide-client-v8
# [feature/MOB-174]  Dark theme dialog in home

**What does this PR do?**

   This PR implements the dark theme dialog in Home. This PR depends on #1267.

**Database changed?**

   No

**How should this be manually tested?**

  Check if the dialog displays on the correct use cases and that the buttons work as they should.
  Possible use cases:
  - Android 10, System Default set to light: should show dialog
  - Version before Android 10: should show dialog
  - It should not show the dialog on other scenarios.

**What are the relevant tickets?**

  [MOB-174](https://aptoide.atlassian.net/browse/MOB-174)

**Code Review Checklist**
